en/config/battery.md

```diff
@@ -4,8 +4,10 @@ This topic explains how to configure power settings so that PX4 can estimate the
 
 ::: info
 These instructions require that the vehicle has a [Power Module (PM)](../power_module/index.md), or other hardware that can measure the battery voltage and (optionally) the current.
+:::
 
-This tuning is not needed for [Smart/MAVLink Batteries](../smart_batteries/index.md).
+::: tip
+This tuning is not needed for [Smart/MAVLink Batteries](../smart_batteries/index.md): batteries that can supply a reliable indication of remaining charge.
```

Comment:
@MaEtUgR Should a battery than can reliably supply discharge rates - max and/or average - do so in [BATTERY_STATUS_V2](https://mavlink.io/en/messages/development.html#BATTERY_STATUS_V2)?

Right now we supply a battery level and we trigger on percentage levels. If we supply a capacity and an average discharge rate then you would have flight time min and average remaining, which would allow a fair bit less guesswork.

Just thinking aloud - perhaps this is overkill. Not going to block merging this.